src/vs/editor/common/commands/trimTrailingWhitespaceCommand.ts

```diff
@@ -15,13 +15,20 @@ export class TrimTrailingWhitespaceCommand implements editorCommon.ICommand {
 
 	private selection: Selection;
 	private selectionId: string;
+	private preserveCursor: boolean;
 
-	constructor(selection: Selection) {
+	constructor(selection: Selection, preserveCursor?: boolean) {
```

Comment:
Shouldn't it be preserving multiple cursors? Not just one?

Reply:
Yeah, I woke up realizing that passing a boolean was silly—I should just provide the cursors directly to the command. Fixed that.